frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPartition.jsx

```diff
@@ -360,19 +363,20 @@ class Column extends React.Component {
       isDragging,
       partitionName,
     } = this.props;
-    const { expanded } = this.state;
+    const { hover, expanded } = this.state;
+    const shadowOffset = hover ? "5px" : "3px";
+    const shadowColor = lighten(colors["text-dark"], hover ? 1.3 : 1.5);
     const showOptionsPanel = expanded && !isDragging;
     return connectDropTarget(
       connectDragSource(
         <div
           className="mb1 bordered rounded"
+          onMouseEnter={() => this.mouseHover(true)}
+          onMouseLeave={() => this.mouseHover(false)}
           style={{
             padding: "12px 14px",
-            "box-shadow": `0 2px 3px ${lighten(colors["text-dark"], 1.5)}`,
-            "&:hover": {
-              "box-shadow": `0 2px 5px ${lighten(colors["text-dark"], 1.3)}`,
-              transition: "all 300ms linear",
-            },
```

Comment:
Why doesn't CSS :hover work anymore?

Reply:
I believe pseudo-classes (like `:hover`) never work with inline styles.

Alternative solution to this mouse enter/leavePR is to create a CSS class. The drawback: the shadow color can't be dynamically computed at run-time. What's your thought @kdoh?

Reply:
This definitely feels like it should be taken care of via `styled-components.` and the native browser `:hover` pseudo selector. Is there a reason we're not using that here and instead doing our own `hover` logic?